src/graph/backend/dnnl/kernels/mqa_decomp_config.cpp

```diff
@@ -52,7 +52,9 @@ bool mqa_decomp_config_t::initial_check(const std::shared_ptr<subgraph_t> &sg,
             batch_size == wei1_user_dims[0] && batch_size == wei2_user_dims[0],
             false,
             "batch size mismatch, batch_size: %lld, wei1: %lld, wei2: %lld",
-            batch_size, wei1_user_dims[0], wei2_user_dims[0]);
+            static_cast<long long int>(batch_size),
+            static_cast<long long int>(wei1_user_dims[0]),
+            static_cast<long long int>(wei2_user_dims[0]));
```

Comment:
I'd say you better stick to a single format like `long int` and `%ld`. Or the other one, but single.